symfony/src/Manager/StatisticsManager.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App\Manager;
+
+use App\Entity\Structure;
+use App\Repository\CampaignRepository;
+use App\Repository\CostRepository;
+use App\Repository\MessageRepository;
+use App\Repository\StructureRepository;
+use App\Repository\VolunteerRepository;
+
+class StatisticsManager
+{
+
+    /**
+     * @var MessageRepository
+     */
+    private $messageRepository;
```

Comment:
As I mentioned in the contributing, a manager can only call its own repository (eg: StatisticsRepository) or managers. The goal is that if we have somehow some business logic to put around the data, we'll do it in the manager and not in the repository and all piece of code using it will benefit of it.

Details here: https://github.com/redcall-io/app/blob/master/CONTRIBUTING.md#code-guidelines

So instead of injecting costRepository, campaignRepository, you should import their Manager instead (costManager, campaignManager), by experience this really helps keeping repositories not containing any business code.